acceptance/tests/resource/file/symbolic_modes.rb

```diff
@@ -1,5 +1,8 @@
 test_name "file resource: symbolic modes"
 confine :except, :platform => /^eos-/ # See ARISTA-37
+confine :except, :platform => /^solaris-10/
+confine :except, :platform => /^windows/
+confine :to, {}, hosts.select { |host| ! host[:roles].include?('master') }
```

Comment:
Nicely done @johnduarte! I wonder if other tests run all hosts (because why not?) when really they should only run on agent hosts?